src/System.Windows.Forms/src/System/Windows/Forms/PictureBox.cs

```diff
@@ -14,39 +15,35 @@
 namespace System.Windows.Forms
 {
     /// <summary>
-    ///  Displays an image that can be a graphic from a bitmap,
-    ///  icon, or metafile, as well as from
-    ///  an enhanced metafile, JPEG, or GIF files.
+    /// Displays an image that can be a graphic from a bitmap, icon, or metafile, as well as from
+    /// an enhanced metafile, JPEG, or GIF files.
     /// </summary>
-    [
-    ComVisible(true),
-    ClassInterface(ClassInterfaceType.AutoDispatch),
-    DefaultProperty(nameof(Image)),
-    DefaultBindingProperty(nameof(Image)),
-    Docking(DockingBehavior.Ask),
-    Designer("System.Windows.Forms.Design.PictureBoxDesigner, " + AssemblyRef.SystemDesign),
-    SRDescription(nameof(SR.DescriptionPictureBox))
-    ]
+    [ComVisible(true)]
+    [ClassInterface(ClassInterfaceType.AutoDispatch)]
+    [DefaultProperty(nameof(Image))]
+    [DefaultBindingProperty(nameof(Image))]
+    [Docking(DockingBehavior.Ask)]
+    [Designer("System.Windows.Forms.Design.PictureBoxDesigner, " + AssemblyRef.SystemDesign)]
+    [SRDescription(nameof(SR.DescriptionPictureBox))]
     public class PictureBox : Control, ISupportInitialize
     {
         /// <summary>
-        ///  The type of border this control will have.
+        /// The type of border this control will have.
         /// </summary>
-        private BorderStyle borderStyle = System.Windows.Forms.BorderStyle.None;
+        private BorderStyle borderStyle = BorderStyle.None;
 
         /// <summary>
-        ///  The image being displayed.
+        /// The image being displayed.
         /// </summary>
         private Image image;
 
         /// <summary>
-        ///  Controls how the image is placed within our bounds, or how we are
-        ///  sized to fit said image.
+        /// Controls how the image is placed within our bounds, or how we are sized to fit said image.
         /// </summary>
         private PictureBoxSizeMode sizeMode = PictureBoxSizeMode.Normal;
         private Size savedSize;
 
-        bool currentlyAnimating;
+        private bool currentlyAnimating;
```

Comment:
We should fix the naming too to align to the guidelines
```suggestion
        private bool _currentlyAnimating;
```

Few other variables as well.